docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
         image: ascent/fluentd:${RELEASE_VERSION:-latest}
         ports:
             - '24224:24224'
+            - '8888:8888'
```

Comment:
We don't want to expose this port outside the container. If its only being used for health checks, we don't need to expose the port at all, since the health check runs inside the container.